packages/theme/src/colors.ts

```diff
@@ -6,13 +6,12 @@ const defaultColorVariant = {
     green10: 'rgba(0, 133, 77, 0.1)',
```

Comment:
please check also "color" colors, I think there is no green20 and 10 anymore and maybe some changes were made to other

Reply:
[fd01348](https://github.com/trezor/trezor-suite/pull/6183/commits/fd01348bbe6329541d752edeb970acce32d0c6f5)